exercises/exercise_2/README.md

```diff
@@ -1,219 +1,195 @@
-words $ echo $?
-1
-```
-* **Fix** `myapp.dockerapp/metadata.yml`
-
-* **Comment** the `services:` line in `myapp.dockerapp/docker-compose.yml`
-* **Validate** the application
-```sh
-words $ docker-app validate myapp
-Error: failed to load composefiles: failed to parse Compose file version: "3.7"
-#services:
-    web:
-     image: dockerdemos/lab-web
-     ports:
-      - "33000:80"
-
-    words:
-     image: dockerdemos/lab-words
-     deploy:
-       replicas: 3
-       endpoint_mode: dnsrr
-
-    db:
-     image: dockerdemos/lab-db
-
-: yaml: line 2: did not find expected key
-
-words $ echo $?
-1
-```
-* **Fix** `myapp.dockerapp/docker-compose.yml`
-
-**Summary**
-* `docker-app` let's you define metadata and parameters on top of a compose file
-* `inspect` displays all important informations in a compact way
-* `validate` checks everything is ok in the application. You can use it as a CI step.
-* `split`/`merge` transforms your application to a multiple-files/one-file. The one-file version is easy to share.
+## Exercise Objective
+
+By the end of this exercise, you will have:
+
+- Created a Docker Application package
+- Learned about the application package components
+- Learned about merging and splitting the components
+- Learned how to inspect and validate the application
+
+
+
+## Docker Application Overview
+
+An application package is a set of 3 documents, plus custom files called `attachments`:
+
+1. A `metadata.yml` file describing the application metadata (name, version, description, ...)
+2. A `docker-compose.yml` file describing the application structure (what we have right now)
+3. A `parameters.yml` file with key/value parameters (we will use this in the next exercise)
+
+
+## Initialize the Application
+
+1. Let's look at the `docker app init` command and see it's options.
+
+    ```bash
+    $ docker app init --help
+
+    Usage:  docker app init APP_NAME [--compose-file COMPOSE_FILE] [--description DESCRIPTION] [--maintainer NAME:EMAIL ...] [OPTIONS]
+
+    Start building a Docker Application package. If there is a docker-compose.yml file in the current directory it will be copied and used.
+
+    Examples:
+    $ docker app init myapp --description "a useful description"
+
+    Options:
+          --compose-file string      Compose file to use as application base (optional)
+          --description string       Human readable description of your application (optional)
+          --maintainer stringArray   Name and email address of person responsible for the application
+                                    (name:email) (optional)
+          --single-file              Create a single-file Docker Application definition
+    ```
+
+2. Let's now create our Docker app! We will use the `docker app init` command and specify the description and maintainer. Feel free to change these values. Be sure to run this in the same directory as the `docker-compose.yml` file we were working on in the last exercise.
```

Comment:
Add "Otherwise you can specify a path to your compose file using `--compose-file path/to/my/docker-compose.yml` ?